### ISSUE TITLE
Default tag does not get overriden.

### ISSUE THREAD

**Issue by EinarasGar** [juhaku/utoipa#955]
Hi, Im trying out version 5.0.0-alpha.0 and have noticed new behavior with tags.
As mentioned in #916 
> This means that the default tag (the handler function name by default) will be added if not overridden with the tag attribute.

However, I do not experience this behavior. 
Using this configuration
```
#[derive(OpenApi)]
#[openapi(
    paths(
        super::handlers::accounts_handler::get_account,
         ...
    ),
```

and 
```
/// Get Account
///
/// Gets a specific account of the user with metadata.
#[utoipa::path(
    get,
    path = "/api/users/:user_id/accounts/:account_id",
    tag = "Accounts",
    responses(
        ...
    ),
    params(
        ...
    ),
    security(
      ...
    )

)]
pub async fn get_account(
```

My openapi spec generates like this
```
"/api/users/:user_id/accounts/:account_id": {
      "get": {
        "tags": [
          "Accounts",
          "super::handlers::accounts_handler"
        ],
        "summary": "Get Account",
        "description": "Gets a specific account of the user with metadata.",
        "operationId": "get_account",
...
```

This results in my redoc view having double of everything.

Reproducable example can be found here
https://github.com/EinarasGar/myra/blob/crate-update/server/api/src/openapi.rs
https://github.com/EinarasGar/myra/blob/crate-update/server/api/src/handlers/accounts_handler.rs

**Comment by NickUfer** [comment]
Got the same problem. You have to add an import statement for what you are using as a path or an api. In your case this should fix it, if super works the same as crate:

```rust
use super::handlers::accounts_handler::get_account;

#[derive(OpenApi)]
#[openapi(
    paths(
        get_account,
         ...
    ),
```

**Comment by eiswind** [comment]
I see the same issue here. But my handler and the #[derive(OpenApi)] are in the same module, so I do not need to import. 
In the generated spec I see my custom tag, as well as the module name.


```rust
#[derive(OpenApi)]
#[openapi(
    tags((name = "Authentication", description = "Authentication operations")),
    paths(authenticate, refresh),
    components(schemas(
        Credentials,
        RefreshCredentials,
        AuthenticationResult
    ))
)]
```

**Comment by eiswind** [comment]
Solved it: Had to import the classes for nest() the same way as described in https://github.com/juhaku/utoipa/issues/955#issuecomment-2155553055

**Comment by juhaku** [comment]
While that works as well, there was a bug in the tags support that caused havoc. This is also elaborated here #978 And should be fixed here #1002.
